Justfile

```
@@ -44,8 +47,8 @@ rootfs $IMAGE: init-work
     set -xeuo pipefail
     ROOTFS="{{ workdir }}/rootfs"
     mkdir -p $ROOTFS
-    ctr="$(sudo podman create --rm "${IMAGE}")" && trap "sudo podman rm $ctr" EXIT
-    sudo podman export $ctr | tar -xf - -C "${ROOTFS}"
+    ctr="$(${SUDOIF} podman create --rm "${IMAGE}" /bin/bash)" && trap "${SUDOIF} podman rm $ctr" EXIT
```

Dont think the /bin/bash is needed, after all, all we do is just create a dummy container to export its filesystem from it
```suggestion
    ctr="$(${SUDOIF} podman create --rm "${IMAGE}")" && trap "${SUDOIF} podman rm $ctr" EXIT
```

---

I added it because although it's not necessary when building locally, it is required when running in GitHub actions. There is the following error without it:
```
++ sudo podman create --rm ghcr.io/ublue-os/bluefin:lts
Error: no command or entrypoint provided, and no CMD or ENTRYPOINT from image
```

- Failed run without `/bin/bash`: https://github.com/ledif/titanoboa/actions/runs/13991440311/job/39176090558
- Completed run with `/bin/bash`: https://github.com/ledif/titanoboa/actions/runs/13991673192